resources/webcourses/src/app/views/default/header-bar/header-bar.component.scss

```diff
@@ -21,6 +21,15 @@
   max-width: 130px;
   overflow: hidden;
   color: darkorange;
+  font-size: large;
+}
+
+#user-email {
+  max-width: 130px;
+  overflow: hidden;
+  color: dimgrey;
+  font-size: x-small;
```

Comment:
Instead of specifying font-size here, let's keep the original `small` tags that were removed in the template file. By keeping as much of the original setup as possible, we reduce code forking. Code forking can make running updates a PITA; million places get flagged with changes after an update, and the review process becomes a long nightmare.

This also reduces cross browser compatibility issues. A group of people tested how `small` tags in this use case affect browsers. We've no idea what font-size: x-small looks like on Safari, if it scales properly, or whether it is even compatible. (I can't find any info on it, but the `xxx-large` property is listed as incompatible with a few mobile browsers, and x-small might be too. [Source](https://caniuse.com/?search=font-size))

Reply:
You cant properly text align in a small tag and this is the only place it was used, simple swapping it to a div just makes sense imo